kornia/losses/focal.py

```diff
@@ -78,11 +80,11 @@ def focal_loss(
         raise ValueError(f"input and target must be in the same device. Got: {input.device} and {target.device}")
 
     # compute softmax over the classes axis
-    input_soft: torch.Tensor = F.softmax(input, dim=1)
-    log_input_soft: torch.Tensor = F.log_softmax(input, dim=1)
+    input_soft: Tensor = F.softmax(input, dim=1)
```

Comment:
```suggestion
    input_soft: Tensor = input.softmax(1)
```
i'd suggest to avoid torch imports so that its easy later to extend to numpy/jax